client/src/ui/molecules/guides-menu/index.tsx

```diff
@@ -4,59 +4,59 @@ import { Menu } from "../menu";
 import "./index.scss";
 
 export const GuidesMenu = ({ visibleSubMenuId, toggleMenu }) => {
-  const locale = useLocale();
+  const locale = useLocale().toLowerCase();
 
   const menu = {
     id: "guides",
     label: "Guides",
-    to: `/${locale}/docs/Learn`,
+    to: `/${locale}/docs/learn`,
```

Comment:
General comment about all the case changes: I think ideally we want to avoid making changes that the mdn/yari maintainer would not merge back into mdn/yari. So if there’s any way we could facilitate static hosting without hardcoding any case changes, I think we should prefer to do that. But if there’s _not_ a way to do that, then I think we should evaluate how important the benefit of static hosting is relative to the cost of making changes that we won’t be able to get the mdn/yari maintainers to merge back into mdn/yari

Reply:
Agreed - plain static file hosting vs. using cloud-function to host files is still a bit of an open discussion ATM